packages/code-du-travail-frontend/src/outils/PreavisRetraite/steps/Result/components/DecryptedResult.tsx

```diff
@@ -102,14 +106,16 @@ const ShowResult: React.FC<ShowResultProps> = ({
 
 type ShowResultAgreementProps = ShowResultProps & {
   isAgreementSupported?: boolean;
+  agreementRoute?: AgreementRoute;
 };
 
 const ShowResultAgreement: React.FC<ShowResultAgreementProps> = ({
   result,
   isAgreementSupported,
   agreementMaximumResult,
+  agreementRoute,
 }) => {
-  if (!result) {
+  if (!result || agreementRoute === "not-selected") {
```

Comment:
pourquoi on a pas juste :
```suggestion
  if (agreementRoute === "not-selected") {
```
?

Reply:
J'ai peur qu'il y ait une regression car c'était : `!result` avant

Reply:
Mais en vrai, ça peut m'aller, car dans tous les cas on a un `return` à la fin